web/package.json

```diff
@@ -33,7 +33,8 @@
     "leaflet-loading": "0.1.24",
     "moment": "2.22.1",
     "uglifyify": "5.0.0",
-    "flatpickr": "4.4.6"
+    "flatpickr": "4.4.6",
+    "leaflet.heightgraph": "0.0.1"
```

Comment:
Oh, interesting. So it is possible to release stuff on npm without any namespace prefix? How does one know that it is original content?

Reply:
> So it is possible to release stuff on npm without any namespace prefix?

Yes, in the beginning npm had no namespace prefix. There is now the option to register for a namespace/organisation. The dependency would then look like `@npmcorp/package-name`.

BTW: we could register an organisation for GH [here](https://www.npmjs.com/org/create) :)

> How does one know that it is original content?

Once you publish a package with a name, only you (or selected contributors) can upload another package with the same name. Usually you could first check the Github repo and check the package.json if the names match.